asciidoc/modules/con_mg_improvements-in-vertx-http-client.adoc

```diff
@@ -32,7 +32,7 @@ Future<HttpClientResponse> f2 = client.post(8080, "localhost", "/uri", HttpHeade
 Future<HttpClientResponse> f3 = client.post(8080, "localhost", "/uri", HttpHeaders.set("foo", "bar"), asyncFile);
 ----
 
-However, you can use the `HttpClientRequest.request()` methods to create an `HttpClientRequest` instance that has not been sent to perform a request operation. These methods can be used in basic interactions such as:
+In {VertX} {v4}, you can use the `HttpClientRequest.request()` methods to create an `HttpClientRequest` instance. These methods can be used in basic interactions such as:
```

Comment:
so here it would rather be: the `requests`methods

Reply:
Done